packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -41,333 +32,320 @@ const DEFAULT_QUERY = {
 	per_page: -1,
 	orderby: 'name',
 	order: 'asc',
-	_fields: 'id,name,parent',
```

Comment:
If these are the only fields used here, you can keep this line, the core-data packages supports the fields config as well.

Reply:
Added _fields.